presto-main/src/main/java/com/facebook/presto/sql/analyzer/SemanticErrorCode.java

```diff
@@ -68,6 +68,8 @@
     VIEW_PARSE_ERROR,
     VIEW_ANALYSIS_ERROR,
     VIEW_IS_STALE,
+    VIEW_CANNOT_BE_RECURSIVE,
```

Comment:
Having two distinct error codes is unnecessary. They both indicate that the given view is recursive.